app/uk/gov/hmrc/brm/connectors/BirthConnector.scala

```diff
@@ -40,35 +40,33 @@ trait BirthConnector {
 
   /** RequestType, reference or details
     */
-  trait RequestType
-
-  case class ReferenceRequest() extends RequestType
-
-  case class DetailsRequest() extends RequestType
+  sealed trait RequestType
+  case object ReferenceRequest extends RequestType
+  case object DetailsRequest extends RequestType
 
   case class Request(uri: String, jsonBody: JsValue)
 
   private def buildRequest(payload: Payload, operation: RequestType): Request = {
     val f       = operation match {
-      case ReferenceRequest() => referenceBody
-      case DetailsRequest()   => detailsBody
+      case ReferenceRequest => referenceBody
+      case DetailsRequest   => detailsBody
     }
     val request = f(payload)
     Request(request._1, request._2)
   }
 
   private def sendRequest(request: Request)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
 
-    val newHc = hc.copy(authorization = None)
+    implicit val headerCarrier: HeaderCarrier = hc.copy(authorization = None)
+    val headersTuple = headers.map(header => (header._1, header._2))
```

Comment:
I dont think you need the `.map(...)` here, I think you can just use `header` as its already a `Seq[(String, String)]`?